infra/cifuzz/cifuzz.py

```diff
@@ -38,27 +38,35 @@
     level=logging.DEBUG)
 
 
-def build_fuzzers(project_name, project_repo_name, commit_sha, git_workspace,
-                  out_dir):
+def build_fuzzers(project_name,
+                  project_repo_name,
+                  workspace,
+                  pr_ref=None,
+                  commit_sha=None):
   """Builds all of the fuzzers for a specific OSS-Fuzz project.
 
   Args:
     project_name: The name of the OSS-Fuzz project being built.
     project_repo_name: The name of the projects repo.
-    commit_sha: The commit SHA to be checked out and fuzzed.
-    git_workspace: The location in the shared volume to store git repos.
-    out_dir: The location in the shared volume to store output artifacts.
+    workspace: The location in a shared volume to store a git repo and build
+      artifacts.
+    pr_ref: The pull request reference to be built.
+    commit_sha: The commit sha for the project to be built at.
 
   Returns:
     True if build succeeded or False on failure.
   """
-  if not os.path.exists(git_workspace):
-    logging.error('Invalid git workspace: %s.', format(git_workspace))
-    return False
-  if not os.path.exists(out_dir):
-    logging.error('Invalid out directory %s.', format(out_dir))
+  # Validate inputs.
+  if not os.path.exists(workspace):
+    logging.error('Invalid workspace: %s.', workspace)
     return False
+  assert pr_ref or commit_sha
```

Comment:
nit: I'd do this before line 59, would make the beginning of the function a bit easier to follow

Reply:
Fixed.